bfcfg

```diff
@@ -53,21 +54,24 @@ mlxmkcap=/lib/firmware/mellanox/boot/capsule/scripts/mlx-mkcap
 
 bfcf_acpi_table=/sys/firmware/acpi/tables/BFCF
 efivars=/sys/firmware/efi/efivars
-sys_cfg_sysfs=${efivars}/BfSysCfg-9c759c02-e5a3-45e4-acfc-c34a500628a6
-redfish_cfg_sysfs=${efivars}/BfRedfish-ce3e5882-6770-4d5e-aa45-90f909da2446
-redfish_state_cfg_sysfs=${efivars}/BfRedfishState-ce3e5882-6770-4d5e-aa45-90f909da2446
-sb_state_sysfs=${efivars}/SecureBoot-8be4df61-93ca-11d2-aa0d-00e098032b8c
-sb_setup_mode_sysfs=${efivars}/SetupMode-8be4df61-93ca-11d2-aa0d-00e098032b8c
 efi_global_var_guid=8be4df61-93ca-11d2-aa0d-00e098032b8c
 efi_bfcfg_var_guid=487ff588-fb71-4b19-9100-ebe067aa1af0
 efi_vlan_var_guid=9e23d768-d2f3-4366-9fc3-3a7aba864374
+efi_redfish_var_guid=ce3e5882-6770-4d5e-aa45-90f909da2446
+efi_debug_var_guid=f3ce977e-c17f-493e-a3cd-5731d386e505
```

Comment:
Q: Are these changes made on top of the latest rebased bfscripts?

Reply:
Yes. The value ce3e5882-6770-4d5e-aa45-90f909da2446 was used in multiple places. This change use a variable instead.